rust/xaynet-server/src/settings.rs

```diff
@@ -566,3 +568,140 @@ where
 
     deserializer.deserialize_str(EnvFilterVisitor)
 }
+
+#[derive(Debug, Deserialize, Validate)]
+#[validate(schema(function = "validate_tls"))]
+/// TLS settings.
+pub struct TlsSettings {
```

Comment:
I would prefer to put these into the api settings.

Reply:
i'll move them.